codacy/requirements-dev.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   alias: minio
 
 - name: rabbitmq-ha
-  version: 1.27.2
+  version: 1.44.3
```

Comment:
matches the version that we was updated already on master on the requirements.yaml